operator/src/main/resources/Operator.properties

```diff
@@ -205,6 +205,9 @@ WLSDO-0012=Managed Server ''{0}'' specified in the domain resource does not exis
 WLSDO-0013=Istio is enabled and the domain resource specified to expose channel {0} in the adminServices section. The channel name default, \
 default-admin, default-secure are internal to the WebLogic Kubernetes Operator. Please create the network channel with a different name in the WebLogic Domain before \
 exposing it.
+WLSDO-0014=The domain {0} introspect job failed, it will automatically retry in {1} seconds. Current number of  \
+  retries = {2}
+WLSDO-0015="Retrying failed domain {0} introspect job, retry {1} of a maximum {2}."
```

Comment:
The maximum doesn't always apply - so maybe the easiest thing to do is remove it?  Sorry if I suggested including it.  Or maybe change maximum --> "potential maximum {2} (the retry limit does not apply to all types of failures).

Reply:
I think it applies.  The domain validation fail case is not in the same code path, it spill out message in the log

WLSKO-0157=Domain {0} is not valid: {1}

Reply:
Ok.